### ISSUE TITLE
Protect/block remote access for clipper service

### ISSUE THREAD

**Issue by diego-betto** [laurent22/joplin#595]
Hi,

if you enable clipper service there is no protection. 
Everybody that knows your IP address can post to your Joplin.
No protections, no password, no origin limitations.

Try this (change ip and port according with your Joplin configuration) from another pc.

`curl -d '{"title":"test remote post"}' -H "Content-Type: application/json" -X POST http://192.168.10.4:41184/notes`

I suggest at least a sort of "Same IP Origin" policy.

Regards,
Diego

**Comment by tessus** [comment]
fixed in b826e2d97bcd69d1b0d2389ce6bf55da825e9c41

**Comment by laurent22** [comment]
Yes the service is now only available to localhost. If you have any suggestion on how to improve it though feel free to let us know.

**Comment by laurent22** [comment]
Yes the service is now only available to localhost (as of the next version). If you have any suggestion on how to improve it though feel free to let us know.

**Comment by diego-betto** [comment]
Tnx a lot!
